<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlurEffectConfig</class>
 <widget class="QWidget" name="BlurEffectConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="minimumHeight">
      <number>25</number>
     </property>
     <widget class="QWidget">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelConstantBlurDescription">
         <property name="text">
          <string>Blur strength:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSlider" name="kcfg_BlurStrength">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TickPosition::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelConstantBlurLight">
             <property name="text">
              <string>Light</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelConstantBlurStrong">
             <property name="text">
              <string>Strong</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelConstantNoiseDescription">
         <property name="text">
          <string>Noise strength:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QSlider" name="kcfg_NoiseStrength">
           <property name="maximum">
            <number>14</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TickPosition::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelConstantNoiseLight">
             <property name="text">
              <string>Light</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelConstantNoiseStrong">
             <property name="text">
              <string>Strong</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelTransparentBlurDescription">
         <property name="text">
          <string>Window opacity affects blur:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="kcfg_TransparentBlur">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelBrightness">
         <property name="text">
          <string>Brightness:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QSlider" name="kcfg_Brightness">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>25</number>
           </property>
           <property name="pageStep">
            <number>25</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TickPosition::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelBrightnessStrong">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelBrightnessNone">
             <property name="text">
              <string>Strong</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelSaturation">
         <property name="text">
          <string>Saturation:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QSlider" name="kcfg_Saturation">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>25</number>
           </property>
           <property name="pageStep">
            <number>25</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TickPosition::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="labelSaturationStrong">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSaturationNone">
             <property name="text">
              <string>Strong</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelContrast">
         <property name="text">
          <string>Contrast:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSlider" name="kcfg_Contrast">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>25</number>
           </property>
           <property name="pageStep">
            <number>25</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TickPosition::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="labelContrastStrong">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelContrastNone">
             <property name="text">
              <string>Strong</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelCornerRadius">
         <property name="text">
          <string>Corner Radius:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QDoubleSpinBox" name="kcfg_CornerRadius"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget">
      <attribute name="title">
       <string>Force blur</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="windowClassesBriefDescription">
             <property name="text">
              <string>Classes of windows to force blur:</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KContextualHelpButton" name="windowClassesContextualHelp">
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="kcfg_WindowClasses">
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutMatchingMode">
         <item>
          <widget class="QRadioButton" name="kcfg_BlurMatching">
           <property name="text">
            <string>Blur only matching</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="kcfg_BlurNonMatching">
           <property name="text">
            <string>Blur all except matching</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_BlurDecorations">
         <property name="text">
          <string>Blur window decorations as well</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_BlurMenus">
         <property name="text">
          <string>Blur menus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_BlurDocks">
         <property name="text">
          <string>Blur docks</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTextBrowser" name="aboutText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Failed to load about.html</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
